File: backend/DTO/Bid/BidHistoryDto.cs
namespace DreamBid.Dtos.Bid
{
    public class BidHistoryDto
    {
        public int AuctionId { get; set; }
        public List<BidDto> Bids { get; set; }
        public decimal HighestBidAmount { get; set; }
        public int TotalBids { get; set; }
    }
}

File: frontend/src/dto/bid/bidHistoryDto.js
class BidHistoryDto {
    auctionId = ""; // Associated auction ID
    bids = []; // List of bids (array of BidDto objects)
    highestBidAmount = ""; // Highest bid amount in the auction
    totalBids = ""; // Total number of bids placed in the auction
    bidTime = ""; // Time of the last bid
}

export default BidHistoryDto;